tools/accuracy_checker/configs/README.md

```diff
@@ -0,0 +1,95 @@
+# How to use predefined configuration files
+
+## Structure
+
+Configuration file declares validation process. Every model has to have entry in `models` list. Each entry has to contain distinct `name`, `launchers` and `datasets` sections.
+
+Example:
+
+```yaml
+models:
+  - name: model_name
+
+    launchers:
+      - framework: dlsdk
+        adapter: adapter_name
+
+    datasets:
+      - name: dataset_name
+```
+
+Also there are composite models, each of which has to have entry in `evaluations` list. Each entry has to contain distinct `name`, `module` and `module_config`. `module_config` has to consist of `network_info`,`launchers` and `datasets` fields.
+
+Example:
+
+```yaml
+evaluations:
+  - name: model_name
+    module: name_of_class_with_custom_evaluators
+    module_config:
+      network_info:
+        encoder: {}
+
+        decoder:
+          adapter: adapter_name
+
+      launchers:
+        - framework: dlsdk
+
+      datasets:
+        - name: dataset_name
+```
+
+## Location
+
+Predefined configuration file `accuracy-check.yml` for each Open Model Zoo model can be found in the model directory.
+
+`model_name.yml` file, which is located in current `configs` folder, is a link to `accuracy-check.yml` for `model_name` model.
+
+Example:
+
+[alexnet.yml](alexnet.yml) is a link for configuration file [accuracy-check.yml](../../../models/public/alexnet/accuracy-check.yml) for `alexnet` model.
+
+## Options
+
+1. To run configuration specify the path to the required configuration file to `-c, --config` command line.
+2. Configuration files don't contain paths to used models and weights. The model and weights are searched automatically by name of model in path specified in `-m, --models` command line option.
+3. The path relative to which the `data_source` is specified can be provided via  `-s, --source` command line. If you want to evaluate models using well-known datasets, you need to organize folders with validation datasets in a certain way. More detailed information about dataset preparation you can find in [Dataset Preparation Guide](../../../datasets.md).
+4. The path relative to which the  `annotation` and `dataset_meta` are specified can be provided via `-a, --annotations` command line. Detailed information about annotation conversion you can find in [Annotation Conversion Guide](../accuracy_checker/annotation_converters/README.md).
```

Comment:
please add info that annotation and dataset_meta (if required) will be stored to this directory if  they do not exist and can be used for the next running to skip annotation conversion step